docs/development/development.mdx

```diff
@@ -119,7 +121,7 @@ The tests are currently always compiled with debugging information enabled, ther
 
 Tests are verified and working with GCC 4.9.3
 
-## Using Git and Github
+## Using Git And Github
```

Comment:
Words like `and` are not capitalized in TitleCase.

What words are not capitalized in Titlecase?
In title case, major words are capitalized and most minor words are lowercase.
...

    short conjunctions (e.g., “and,” “as,” “but,” “for,” “if,” “nor,” “or,” “so,” “yet”)
    articles (“a,” “an,” “the”)
    short prepositions (e.g., “as,” “at,” “by,” “for,” “in,” “of,” “off,” “on,” “per,” “to,” “up,” “via”)

Reply:
`check_title_case.mjs` was complaining the whole time but I don't know what was wrong...

Reply:
Yes it's not optimal (as in didn't look deeper), have been fighting it a lot as it might not take above rules into account.

My problem is when following README `npm start` produces some files blocking the git commit. So using two terminals to workaround that :roll_eyes:

Reply:
I'll see what I can do